notebooks/18_deep_neural_network/index.md

```diff
@@ -0,0 +1,201 @@
+## Fully Connected Layer
+
+Fully Connected layers in neural networks are those layers where all the inputs from one layer are connected to every activation unit of the next layer. In most popular CNN models, the last few layers are fully connected layers that compile the data extracted by previous layers to form the final output. It is the second most time-consuming layer second to the Convolution Layer.
+![](https://i.imgur.com/3nItEgk.png)
+
+## Conv Layer
+This layer is the main difference between CNNs and MLPs. Convolution in the word refers to two operators between two functions. In mathematics convolution define as below:  
+<img src="https://render.githubusercontent.com/render/math?math=(f * g)(t) := \int_{-\infty}^{\infty} f(\tau)g(t - \tau)d\tau">  
+In this image, you can see an example for functions of convolution and two other similar operators.
+    
+<figure>
+  <img
+  src="https://upload.wikimedia.org/wikipedia/commons/thumb/2/21/Comparison_convolution_correlation.svg/800px-Comparison_convolution_correlation.svg.png"
+  alt="">
+  <center><figcaption></figcaption>
+</figure>
+
+
+
+Here we’ll not talk about details, but convolutional layers are somehow enabling convolution operator on sub-matrices of the image. These layers have formed from some kernel with the same height, width, and depth. The number of these kernels is equal to the depth of the output. Also, the depth of each kernel must be equal to the depth of input. For example, if you have RGB data, your first convolutional layer kernels depth must be 3.  
+In the context of a convolutional neural network, convolution is a linear operation that involves the multiplication of a set of weights with the input. A convolution layer has formed by 1 or more of these operations that each of them called a kernel. All kernels have the same height, width, and depth. To find the output of the layer, we put the first kernel on the top-right of the input and calculate the output of the kernel, and put it as the first cell of a matrix. After that, we move it to right and calculate again, and put the result in the second cell. When we receive to end of columns, we move the kernel down. we do this until we rich to the end of the image. We do this for all kernels and this is how we make the output of the convolutional layer.
+    
+<figure>
+  <center><img
+  src="https://miro.medium.com/max/1070/1*Zx-ZMLKab7VOCQTxdZ1OAw.gif"
+  alt="">
+  <center><figcaption></figcaption>
+</figure>
+<figure>
+  <center><img
+  src="https://miro.medium.com/max/738/1*Q7NXeOlDkm4xlNrNQOS67g.gif"
+  alt="">
+  <center><figcaption></figcaption>
+</figure>
+
+
+
+## Pooling
+
+Similar to the Convolutional Layer, the Pooling layer is responsible for reducing the spatial size of the Convolved Feature.
+While a lot of information is lost in the pooling layer, it also has a number of benefits to the Convolutional neural network. They help to reduce complexity, improve efficiency, and limit the risk of overfitting.
```

Comment:
if you want to use capital case, use it for all words of a phrase: Convolutional Neural Network